node/libs/crypto/src/secp256k1/testonly.rs

```diff
@@ -0,0 +1,62 @@
+use crate::secp256k1::{AggregateSignature, PublicKey, SecretKey, Signature};
+use elliptic_curve::ScalarPrimitive;
+use k256::ecdsa::RecoveryId;
+use rand::{
+    distributions::{Distribution, Standard},
+    CryptoRng, Rng, RngCore,
+};
+
+struct RngWrapper<R>(R);
+
+impl<R: Rng> CryptoRng for RngWrapper<R> {}
+impl<R: Rng> RngCore for RngWrapper<R> {
+    fn next_u32(&mut self) -> u32 {
+        self.0.next_u32()
+    }
+
+    fn next_u64(&mut self) -> u64 {
+        self.0.next_u64()
+    }
+
+    fn fill_bytes(&mut self, dest: &mut [u8]) {
+        self.0.fill_bytes(dest)
+    }
+
+    fn try_fill_bytes(&mut self, dest: &mut [u8]) -> Result<(), rand::Error> {
+        self.0.try_fill_bytes(dest)
+    }
+}
```

Comment:
do we need these? For bn254 we needed them, because it was using an older version of rand.

Reply:
`k256::SecretKey::random` required `CryptoRngCore`